libminifi/include/core/state/nodes/DeviceInformation.h

```diff
@@ -468,6 +470,26 @@ class DeviceInfoNode : public DeviceInformation {
 
  protected:
 
+  /**
+   * Have found various ways of identifying different operating system variants
+   * so these were either pulled from header files or online.
+   */
+  static inline std::string getOperatingSystem() {
+#ifdef _WIN32
+    return "Windows 32";
+#elif _WIN64
+    return "Windows 64";
```

Comment:
This branch is dead in my opinion. I think _WIN32 is always defined when _WIN64 is, but I might be wrong.

Reply:
hmm, that's a good point. I'll make that a conditional Or. We can make them both windows. Arch, elsewhere sort of provides this already.